tests/FSharp.Data.GraphQL.IntegrationTests.Server/Schema.fs

```diff
@@ -130,56 +136,27 @@ module Schema =
+
+        Define.Object<Root> (
+            name = "MutationType",
             fields =
-                [ Define.Field(
-                    name = "singleUpload",
-                    typedef = UploadedFileType,
-                    description = "Uploads a single file to the server and get it back.",
-                    args = [ Define.Input("file", Upload, description = "The file to be uploaded.") ],
-                    resolve = fun ctx _ -> mapUploadToOutput (ctx.Arg("file")))
-                  Define.Field(
-                    name = "nullableSingleUpload",
-                    typedef = StructNullable UploadedFileType,
-                    description = "Uploads (maybe) a single file to the server and get it back (maybe).",
-                    args = [ Define.Input("file", Nullable Upload, description = "The file to be uploaded.") ],
-                    resolve = fun ctx _ -> ctx.TryArg("file") |> ValueOption.flatten |> ValueOption.map mapUploadToOutput)
-                  Define.Field(
-                    name = "multipleUpload",
-                    typedef = ListOf UploadedFileType,
-                    description = "Uploads a list of files to the server and get them back.",
-                    args = [ Define.Input("files", ListOf Upload, description = "The files to upload.") ],
-                    resolve = fun ctx _ -> ctx.Arg("files") |> Seq.map mapUploadToOutput)
-                  Define.Field(
-                    name = "nullableMultipleUpload",
-                    typedef = StructNullable (ListOf UploadedFileType),
-                    description = "Uploads (maybe) a list of files to the server and get them back (maybe).",
-                    args = [ Define.Input("files", Nullable (ListOf Upload), description = "The files to upload.") ],
-                    resolve = fun ctx _ -> ctx.TryArg("files") |> ValueOption.flatten |> ValueOption.map (Seq.map mapUploadToOutput))
-                  Define.Field(
-                    name = "nullableMultipleNullableUpload",
-                    typedef = StructNullable (ListOf (Nullable UploadedFileType)),
-                    description = "Uploads (maybe) a list of files (maybe) to the server and get them back (maybe).",
-                    args = [ Define.Input("files", Nullable (ListOf (Nullable Upload)), description = "The files to upload.") ],
-                    resolve = fun ctx _ -> ctx.TryArg("files") |> ValueOption.flatten |> ValueOption.map (Seq.map (Option.map mapUploadToOutput)))
-                  Define.Field(
-                    name = "uploadRequest",
-                    typedef = UploadResponseType,
-                    description = "Upload several files in different forms.",
-                    args = [ Define.Input("request", UploadRequestType, description = "The request for uploading several files in different forms.") ],
-                    resolve = fun ctx _ -> mapUploadRequestToOutput (ctx.Arg("request"))) ])
```

Comment:
Can you comment just this block instead of deleting?

By the way, we need to test nullable scalar and nullable field in the input object in our unit tests too.
And test 2 files upload. As there may be an issue.